src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -4836,7 +4832,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.7;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.5;
```

Comment:
I think this should stay at 13.7 for the 1.11 release

Reply:
Thanks. I forked from 1.12 so it was obviously 12.5 there.